webapp/src/components/modals/channel_subscriptions/edit_channel_subscription.tsx

```diff
@@ -244,10 +244,11 @@ export default class EditChannelSubscription extends PureComponent<Props, State>
         }
 
         this.setState({instanceID, error: null});
-        this.handleProjectChange('');
+        this.handleProjectChange({});
     }
 
-    handleProjectChange = (projectID: string) => {
+    handleProjectChange = (fieldValues: DefaultFieldValues) => {
+        const projectID = fieldValues.project_key ? fieldValues.project_key : '';
```

Comment:
```suggestion
        const projectID = fieldValues.project_key ?? '';
```

Reply:
check if this works. fix everywhere if updating

Reply:
We do not have the setup for nullish coleasing in Jira